### PR TITLE
feat/rosetta sdk version upgrade v0.6.10

### PR DESCRIPTION
**Pull request by noandrea**
related to #137

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,22 +1,11 @@
...
-crg: go.sum
-	go build -mod=readonly ./cmd/crg/
-
-gen-all: gen-mocks
-
-gen-mocks:
-	scripts/gen-mocks
-
-install:
-	go install ./cmd/crg
```
Comment (alessio) [inline Makefile:12]: Why did you remove the `crg` and `install` targets?

**Review by jgimeno — APPROVED** [review]
